src/material-examples/expansion-steps/expansion-steps-example.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+
+
+@Component({
+  selector: 'expansion-overview-example',
+  templateUrl: 'expansion-overview-example.html',
+})
+export class ExpansionStepsExample {
+  public step = 0;
```

Comment:
Nope, in fact our [coding standards](https://github.com/angular/material2/blob/master/CODING_STANDARDS.md#access-modifiers) specifically say to omit it.